messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/DiscoveryGroupAdd.java

```diff
@@ -63,15 +67,38 @@ private DiscoveryGroupAdd() {
         super(DiscoveryGroupDefinition.ATTRIBUTES);
     }
 
+    @Override
+    public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+        // Handle jgroups-channel -> jgroups-cluster rename
+        if (!operation.hasDefined(CommonAttributes.JGROUPS_CLUSTER.getName()) && operation.hasDefined(CommonAttributes.LEGACY_JGROUPS_CLUSTER.getName())) {
```

Comment:
Does this code duplicate BroadcastGroupAdd ?
It that's the case, we could put in it in a single place.

Reply:
Just the 1st if block is the same for both add operation handlers, so yes I can consolidate this.